source/Calamari.Azure/Scripts/DeployAzureCloudService.ps1

```diff
@@ -51,8 +51,12 @@ function CreateNewDeployment()
 function WaitForComplete() 
 {
     $completeDeployment = Get-AzureDeployment -ServiceName $OctopusAzureServiceName -Slot $OctopusAzureSlot
-
     $completeDeploymentID = $completeDeployment.DeploymentId
+    $completeDeploymentUrl = $completeDeployment.Url
+        
+    Set-OctopusVariable -name "AzureDeploymentID" -value $completeDeploymentID
+    Set-OctopusVariable -name "AzureDeploymentUrl" -value $completeDeploymentUrl

```

Comment:
Can you name these output variables in the following format: `OctopusAzureCloudServiceDeployment*`. Makes it less of a chance for a possible naming clash later on.